### PR TITLE
adds toString() for pretty printing

### PR DESCRIPTION
**Pull request by leshi**

### CODE REVIEW

File: u2f-ref-code/java/src/com/google/u2f/server/impl/attestation/android/AuthorizationList.java [hunk 1]
```diff
@@ -23,6 +23,10 @@ public Algorithm getAlgorithm() {
     return algorithm;
   }
 
+  public boolean isEmpty() {
```
Comment (juanlang) [inline u2f-ref-code/java/src/com/google/u2f/server/impl/attestation/android/AuthorizationList.java]: Appears unused?

Comment (leshi) [inline u2f-ref-code/java/src/com/google/u2f/server/impl/attestation/android/AuthorizationList.java]: This change is split out from a bigger change (trying to make smaller pull requests).  I will use this method there (or remove it).

Comment (juanlang) [inline u2f-ref-code/java/src/com/google/u2f/server/impl/attestation/android/AuthorizationList.java]: Seems funny to add dead code.. something to keep an eye on, I won't fuss about it here.

**Comment by juanlang** [comment]
lgtm

**Pull request merged**